virttest/qemu_vm.py

```diff
@@ -2245,8 +2254,14 @@ def sort_key(dev):
             if balloon_params.get("balloon_dev_add_bus") == "yes":
                 balloon_bus = _get_pci_bus(
                     devices, balloon_params, 'balloon', True)
+            kwargs = {}
+            if balloon_params.get("deflate-on-oom") == "yes":
+                kwargs["deflate-on-oom"] = "yes"
+            if balloon_params.get("guest-polling"):
+                val = balloon_params.get("guest-polling", 0)
+                kwargs["guest-stats-polling-interval"] = val
```

Comment:
In order to run L2261 and L2262,   balloon_params.get("guest-polling") can't be "0" in L2260.
But you set the default value for "guest-polling" to "0" in L2261. it is unnecessary from my understanding.

Does following code meet what you want to do ?
```
kwargs["guest-stats-polling-interval"] = balloon_params.get("guest-polling", 0)
```

Reply:
@yanan-fu This part is to complete the qemu cmdline. The default value for option "guest-stats-polling-interval" is 0 if it is not specified in cmdline. In another word, if the value is 0, it doesn't matter whether to add it in cmdline.

Reply:
@huangyum  This match with my previous understanding.  If so, one line is ok, or no need to set default value in L2261.